src/main/java/picard/fingerprint/CrosscheckMetric.java

```diff
@@ -59,51 +59,54 @@ public Boolean isMatch() {
         }
     }
 
+    /** The data type. */
```

Comment:
These descriptions should probably be expanded, I just changed them to Javadoc and fixed up a few LEFT vs. RIGHT typos.